customer-samples/RabbitMQ.Sample/Send/Send.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Datadog.Trace;
+using RabbitMQ.Client;
+
+namespace Send
+{
+    class Send
+    {
+        // This small application follows the RabbitMQ tutorial at https://www.rabbitmq.com/tutorials/tutorial-one-dotnet.html
+        public static void Main(string[] args)
+        {
+            var factory = new ConnectionFactory() { HostName = "localhost" };
+            using (var connection = factory.CreateConnection())
+            {
+                using (var channel = connection.CreateModel())
+                {
+                    channel.QueueDeclare(queue: "hello",
+                                         durable: false,
+                                         exclusive: false,
+                                         autoDelete: false,
+                                         arguments: null);
+
+                    // Start a new Datadog span
+                    using (var scope = Tracer.Instance.StartActive("rabbitmq-publish"))
+                    {
+                        string message = "Hello World!";
+                        var body = Encoding.UTF8.GetBytes(message);
+
+                        // Create BasicProperties and a Headers dictionary to store header information
+                        var properties = channel.CreateBasicProperties();
+                        properties.Headers = new Dictionary<string, object>();
+
+                        // Get properties for the active Datadog span
+                        ulong traceId = scope.Span.TraceId;
+                        ulong spanId = scope.Span.SpanId;
+                        string samplingPriority = scope.Span.GetTag(Tags.SamplingPriority);
+
+                        // Add properties to the Headers dictionary in the following way:
+                        //  - "x-datadog-trace-id": "<trace_id>"
+                        //  - "x-dataadog-parent-id": "<span_id>"
+                        //  - "x-datadog-sampling-priority": "<sampling_priority>"
+                        properties.Headers.Add(HttpHeaderNames.TraceId, traceId.ToString(CultureInfo.InvariantCulture));
```

Comment:
I noticed the header are `object` references, but you treat them as `byte[]` in the receiver. Do we know how they are getting converted into UTF8 bytes? What happens if we pass a `byte[]` instead of a `string`?

I'm thinking if RabbitMQ just sends the raw bytes across, using `BitConverter` to convert the header values to/from raw bytes may make the code simpler than using strings as an intermediate format.

Reply:
@lucaspimentel Apparently `string` header values are automatically converted into `byte[]` by RabbitMQ. `byte[]` header values remain intact, so yes it might be clearer to just use `BitConverter` to send these. I've implemented that in the latest commit ecedda54f7c6717d1f804f8b55350e67af8befcf